protostar/configuration_file/configuration_file.py

```diff
@@ -86,6 +86,17 @@ def set_command_names_provider(
     ):
         pass
 
+    @staticmethod
+    def create_appending_cairo_path_suggestion() -> str:
+        return "\n".join(
+            [
+                "You may want to add your new library's path to the 'cairo_path' "
+                "section of your configuration file (usually `protostar.toml`).",
```

Comment:
`cairo_path` isn't a section. `cairo-path` is an attribute.

> "You may want to update your `cairo-path` configuration in {configuration_file_name}."
> "For more information ..."

Reply:
Ok, in your other comment you said

> We should either print the exact name or remove the guess.

So, with the "guess" removed, we can just address the config file in general as "your config file", right?

Reply:
[update msg](https://github.com/software-mansion/protostar/pull/981/commits/a998dc15508fe8fab245e741553b08787011c826)